tests/end-to-end/external-solution-integration/internal/scenario/compass/compass.go

```diff
@@ -67,6 +67,7 @@ func (s *Scenario) Steps(config *rest.Config) ([]step.Step, error) {
 			knativeEventingClientset.EventingV1alpha1().Brokers(s.testID), knativeEventingClientset.MessagingV1alpha1().Subscriptions(helpers.KymaIntegrationNamespace)),
 		testsuite.NewCreateKnativeTrigger(s.testID, helpers.DefaultBrokerName, functionEndpoint, knativeEventingClientset.EventingV1alpha1().Triggers(s.testID)),
 		testsuite.NewSendEventToMesh(s.testID, helpers.FunctionPayload, state),
+		testsuite.NewSleep(s.waitTime),
```

Comment:
Shouln't both sleeps be at the same position? So either both before or both after `NewSendEventToMesh`?

Reply:
Done